src/pic_out.f90

```diff
@@ -1803,11 +1803,8 @@ subroutine part_pdata_out(tnow,xmin_out,xmax_out,ymax_out,pid,jmp)
    nx_loc,ny_loc,nz_loc,0,0/)
 
  write(fname,'(a6,i2.2)')part(pid),iout      !serve sempre
- write(fnamel,'(a6,i2.2,a1,i3.3)')part(pid),iout,'_',imodz !usare con mpi_write_part_col
```

Comment:
Is it necessary?

Reply:
Restored mpi_write_part_col. Its deletion was a mistake. Same for diag_part_dist.